### PR TITLE
FE-1642: Add CSP header

### PR DESCRIPTION
**Pull request by theus**
### What Changed

- Add CSP Headers

### How To Test or Verify

- Verify if the CSP header are present on preview build

#### All PRs Checklist

- [x] Give your pull request a meaningful name.
- [x] Use lowercase filenames.
- [x] Apply at least one team label according to which team is the content expert (ie. `team-FE` or `team-SAZ`)
- [ ] Pull request approval from the FE team or content experts (see label applied above) that isn't the content creator.

### CODE REVIEW

**Comment by theus** [comment]
https://deploy-preview-673--support-docs.netlify.app/

**Comment by shrirupa** [comment]
came across this in preview
`The Content Security Policy directive 'upgrade-insecure-requests' should be empty, but was delivered with a value of '1'. The directive has been applied, and the value ignored.`

File: content/docs/tech-resources/webhook-event-reference.md [hunk 1]
```diff
@@ -12,26 +12,12 @@ Additional resources: [Webhook Authentication](https://www.sparkpost.com/docs/te
 
 ## Event Types
 
-Events which are sent from SparkPost webhooks are organized into different event types. Below is a table that provides you with a list of these event types and map containing each event which fall into their respective event types.
-
-<div id="content-event-types" style="margin: 0 1.5rem"></div>
-<script id="event-type-template" type="text/x-handlebars-template"><table id="event-types"> <thead> <tr> <td>Event Type</td> <td>JSON Key</td> <td>Description</td> <td>Events</td> </tr> </thead> <tbody> {{#each this}} <tr> <td>{{ name }}</td> <td>{{ key }}</td> <td>{{ description }}</td> <td>{{ events }}</td> </tr> {{/each}} </tbody> </table></script>
+Events which are sent from SparkPost webhooks are organized into different event types. Below is a table that provides you with a [list of these event types](https://developers.sparkpost.com/api/webhooks/#header-webhook-event-types) and map containing each event which fall into their respective event types.
```
Comment (shrirupa) [inline content/docs/tech-resources/webhook-event-reference.md:15]: why did we change this ?

Comment (shrirupa) [reply content/docs/tech-resources/webhook-event-reference.md:15]: ohh I found the backstory. https://messagebird.slack.com/archives/C02PVLZGF47/p1650640315695029?thread_ts=1650637243.965329&cid=C02PVLZGF47

Comment (theus) [reply content/docs/tech-resources/webhook-event-reference.md:15]: Sorry, forgot to mention it 🤦‍♂️

**Review by shrirupa — APPROVED** [review]

**Review by shrirupa — APPROVED** [review]

**Pull request merged**